### PR TITLE
RUN-3727 experimental application option

### PR DESCRIPTION
**Pull request by StevenEBarbaro**
experimental application option for shimming `--disable-init-reload` per app (process)

```
'experimental': {
  'disableInitialReload': false,
  'v2Api': false
},
```

Relates to https://github.com/openfin/runtime/pull/783

main.js already has the correct spelling
https://github.com/HadoukenIO/core/blob/develop/src/renderer/main.js#L48

### CODE REVIEW

File: src/browser/convert_options.js [hunk 1]
```diff
@@ -239,6 +240,12 @@ module.exports = {
             plugins: newOptions.plugins
         };
 
+        // v2Api Fallback for incorrect spelling
```
Comment (HarsimranSingh) [inline src/browser/convert_options.js]: Is there a reason to support the misspelling? This feature is going out in `8.56.27.*`, which is still in staging. Why don't we fix it there?

Comment (StevenEBarbaro) [reply src/browser/convert_options.js]: I support that 👍

File: src/browser/convert_options.js [hunk 2]
```diff
@@ -239,6 +240,8 @@ module.exports = {
             plugins: newOptions.plugins
         };
 
+        newOptions.webPreferences.disableInitialReload = newOptions.experimental.disableInitialReload;
```
Comment (HarsimranSingh) [inline src/browser/convert_options.js]: Move this into https://github.com/HadoukenIO/core/pull/277/files#diff-bb76f534ea36dcd8884c45b25ab0f7b0R238

**Comment by HarsimranSingh** [comment]
@StevenEBarbaro can we get more of a description of what this does and also does this need to be documented?

**Pull request merged**